src/Build/Evaluation/ProjectRootElementCache.cs

```diff
@@ -208,7 +208,7 @@ internal ProjectRootElementCache(bool autoReloadFromDisk)
 
                 if (projectRootElement != null && _autoReloadFromDisk)
                 {
-                    FileInfo fileInfo = FileUtilities.GetFileInfoNoThrow(projectFile);
+                    var fileInfo = FileUtilities.GetFileInfoNoThrow(projectFile);
```

Comment:
I'm a bit skeptical of this change; isn't our style generally "use `var` only when the type is unambiguous, usually because it's a constructor on the right-hand side"? Or will this help with a substitution later?

Reply:
Hmm. On the one hand this is nice and easy. On the other, if it were harder to do that, it might be easier to see the impact of changes in PR? I'm fine leaving the change in.